ArmstrongPy.py

```diff
@@ -1,4 +1,5 @@
 # Python program to check if the number is an Armstrong number or not
+
 num = int(input("Enter a number: "))
 sum = 0
 
```

Comment:
Try to use method, function or lamda fuction for writing a program

Reply:
Kindly review the changes.... @amitsahu8 @rohitlamsonge66